src/components/flowchart/styles/_node.scss

```diff
@@ -4,15 +4,13 @@
   text {
     font-weight: 600;
     font-size: 1.4em;
-    transition: opacity ease 0.3s 0.2s;
     pointer-events: none;
 
     .kui-theme--light & {
       fill: $node-labeltext-fill-light;
     }
 
     .kui-theme--dark & {
-      text-shadow: 0 0 6px black;
```

Comment:
I am a little disappointed to remove this because I did think it helped legibility, but if it hurt performance then I think this is a good call